src/kraken/std/docker/kaniko.py

```diff
@@ -75,6 +76,8 @@ def get_kaniko_executor_command(self, dockerfile: str | None, tar_path: str | No
             executor_command += ["--single-snapshot"]
         if self.kaniko_cache_copy_layers.get():
             executor_command += ["--cache-copy-layers"]
+        if self.kaniko_use_compressed_caching.get():
```

Comment:
I think this would need to be `if not ...`?

```suggestion
        if not self.kaniko_use_compressed_caching.get():
```

Also begs the question whether we want to rely on the default in Kaniko to use compressed caching forever. What you could do is not give it a default in line 28 and then

```py
compressed = self.kaniko_use_compressed_caching.get_or(None)
if compressed is not None:
    executor_command += ["--compressed-caching=" + str(bool(compressed)).lower()]
```

Reply:
That's fair, but it's done with other properties so I feel it's acceptable? (cache copy layers for example)